applications/convection_diffusion_application/tests/cpp_tests/test_example.cpp

```diff
@@ -0,0 +1,39 @@
+// KRATOS ___ ___  _  ___   __   ___ ___ ___ ___ 
```

Comment:
why did you add this file?

Reply:
To check that the cpp test compiled on the app, can be removed